state/multiwatcher.go

```diff
@@ -183,6 +184,15 @@ func newStoreManagerNoRun(backing Backing) *storeManager {
 	}
 }
 
+// newDeadStoreManager returns a store manager instance
+// that is already dead and always returns the given error.
+func newDeadStoreManager(err error) *storeManager {
+	var m storeManager
+	m.tomb.Kill(errors.Trace(err))
```

Comment:
ISTM that the callers are passing in errors.Trace(err)
So we need to just use Kill(err) here?